project-base/storefront/components/Forms/validationRules.ts

```diff
@@ -163,37 +163,30 @@ export const validatePassword = (t: Translate): BaseSchema => {
         );
 };
 
-export const validateRepeatedPassword = (t: Translate): BaseSchema => {
+const passwordConfirmValidationSchema = (t: Translate, fieldName: string, requiredMessage: string) => {
```

Comment:
Please, rename `fieldName` to something more descriptive. I suppose this is the first password's field name, so name it as such. This could be `passwordFieldName` or similar.